src/main/java/org/jenkinsci/remoting/protocol/impl/BIONetworkLayer.java

```diff
@@ -273,6 +287,13 @@ public void run() {
                 } finally {
                     release(buffer);
                 }
+            } catch (Throwable e) {
+                if (LOGGER.isLoggable(Level.SEVERE)) {
```

Comment:
We're not exactly on the critical path here... and if we decide to change the level it makes it easier to change it later